image/image.go

```diff
@@ -26,6 +26,9 @@ import (
 // validates the manifest pointed to by the given ref
 // or returns an error if the validation failed.
 func ValidateLayout(src, ref string) error {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
> If other functions work out to remove the checking, it is hard to be aware, and the bug will be recurrent.

This argument doesn't hold up; the "other functions" (in this case the validation code) absolutely need to be able to deal with such a case - if they can't, then it's a serious bug that we shouldn't be trying to mask here. 

If you want to make the argument that it's cleaner/more consistent for end users, that's a different story